### DISCUSSION TITLE
More error message improvements

### DISCUSSION THREAD

**Discussion by JessRynkar** [payloadcms/payload#2537] [Feature Requests & Ideas]
> MissingFieldType: Field is either missing a field type or it does not match an available field type

Would be helpful to state which field is causing the error.

Related discord: https://discord.com/channels/967097582721572934/1091413628856569898

**Comment by AlessioGr** [comment]
This would also be great for this error:

`Collection "xxx" > Field "xxx" > "value" does not match any of the allowed types`

**Reply by JessRynkar** [reply]
@AlessioGr 💯 

I'll change this to a general 'improve error messaging' so we can tack on any others that need it.

**Comment by 60pfennig** [comment]
I want to push this feature request. Error messages in Payload are mostly not helpfull. This is true for field validation errors, where at least the field value which is invalid should be logged together with a specific validation error, as well as misconfiguration errors of the payload config (for example you set a block slug and a collection slug the same the resulting error message is totally misleading). Please take this serious because for me spended the the last 4 weeks developing a payload app this has be a major pain point and it has lowered the dev experience quite a bit.

**Reply by designly1** [reply]
bump
